src/components/TypeBadge/index.jsx

```diff
@@ -1,6 +1,6 @@
-const TypeBadge = ({jobType}) => {
+const TypeBadge = ({jobType, bgColor = '', textColor = ''}) => {
```

Comment:
Having to pass in the bgColor and textColor classes isn't the most intuitive way to use this. Imagine you're a developer working on this code, and you want to add a `TypeBadge` to your page. How would you work out what prop values to pass in for the `bgColor` and `textColor`? Better to have all the styling self-contained to the component and maybe toggled with one prop.

Reply:
Design was for the component to be generic and developer passes the appropriate styling via prop as set forth via the product design and wireframe contained within stories. Will amend design decisions in future projects.